typescript/infra/scripts/check-deploy.ts

```diff
@@ -153,18 +154,21 @@ async function check() {
       throw new Error('Warp route id required for warp module');
     }
     const config = await getWarpConfig(multiProvider, envConfig, warpRouteId);
-    const addresses = getWarpAddresses(warpRouteId);
-    const filteredAddresses = Object.keys(addresses) // filter out changes not in config
+    const warpAddresses = getWarpAddresses(warpRouteId);
+    const filteredAddresses = Object.keys(warpAddresses) // filter out changes not in config
       .filter((key) => key in config)
       .reduce((obj, key) => {
-        obj[key] = addresses[key];
+        obj[key] = {
+          ...warpAddresses[key],
+          proxyAdmin: chainAddresses[key].proxyAdmin,
```

Comment:
this assumes that the proxyAdmin of the mailbox should always be the same as warp routes, is that desirable? (I think so)

Reply:
We have taken that stance, one proxyAdmin per chain that owns all proxies, we've changed the admin contract for proxies that did not meet this requirement